queries/javascript/jsx.scm

```diff
@@ -9,4 +9,4 @@
 (jsx_opening_element name: (identifier) @variable.builtin)
 (jsx_self_closing_element name: (identifier) @variable.builtin)
 
-(jsx_text) @none
+(jsx_text) @Normal
```

Comment:
These feel a little weird being the only ones uppercase, but I don't have a better suggestion, maybe they are just fine, but we should document that uppercase values will map to default hl groups?

Reply:
This is documented upstream.

Reply:
the issue mapping stuff to Normal is that it will conflict with the CursorLine (because Normal applies a background colors too), i would prefer not matching that.

Reply:
Also javascript inside jsx_text might not be properly highlighted.
```javascript
<div>some {someJsFunction(withSomeJsVariable)} {`and maybe strings too ${withSomeVariables}`}</div>
```